### DISCUSSION TITLE
Submit to Journal of Open Source Software (JOSS)

### DISCUSSION THREAD

**Discussion by ketozhang** [rpoleski/MulensModel#61] [General]
Should or is MulensModel planning to be submitted to [JOSS](https://joss.theoj.org/about)?

**Comment by rpoleski** [comment]
We have considered that some time ago. It has definite advantages (e.g., it's likely we'll get valuable feedback), but decided that at that point we prefer a single paper with higher visibility than two papers. Of course we'll change the decision at some point. 
If you want credit for your work, please contact us directly and we'll try to find the solution.

**Comment by ketozhang** [comment]
Understood. It's a journal that gets brought up more frequently in ADASS and ASCL seems happy about it. 

I appreciate the credit, just being listed in the repo is well-appropriate for the portion I've contributed.
